app/src/main/java/org/jellyfin/androidtv/preference/store/DisplayPreferencesStore.kt

```
@@ -72,6 +72,9 @@ abstract class DisplayPreferencesStore(
 				displayPreferencesDto = DisplayPreferencesDto.empty()
 			}
 
+			// TMP for reproduction
+			throw err
```

This is to make sure the app doesn't ignore the exception but crashes instead